client/object_put.go

```diff
@@ -295,12 +294,12 @@ func (x *payloadWriter) Write(p []byte) (int, error) {
 }
 
 func (x *payloadWriter) Close() error {
-	res, err := x.stream.Close()
+	_, err := x.stream.Close()
 	if err != nil {
 		return err
 	}
 
-	return apistatus.ErrFromStatus(res)
+	return errors.New("unexpected error")
```

Comment:
This one looks wrong to me (unlike `InitDataStream`), here the function should be able to `Close()` successfully.

Reply:
Yes, it was an error to replace this place